datasets/sem_eval_2020_task_11/sem_eval_2020_task_11.py

```diff
@@ -127,6 +129,12 @@ def _split_generators(self, dl_manager):
         """Returns SplitGenerators."""
         path_to_manual_file = os.path.abspath(os.path.expanduser(dl_manager.manual_dir))
 
+        tc_labels_template = os.path.join(path_to_manual_file, "test-task-tc-template.out")
+        if os.path.isfile(tc_labels_template):
+            tc_test_template = self._process_tc_labels_template(tc_labels_template)
+        else:
+            tc_test_template = None
```

Comment:
Out of curiosity, why do you need to test if the file exists ?

Reply:
This is a classification task on a span in a given article. So this template just provides the offsets for the spans to be classified in an article.

Reply:
I see.
And in these lines you're checking if the file with the template exists.
Is this verification necessary ? Isn't the file always present in the `ptc-corpus.tgz` file downloaded for this dataset ?

Reply:
For reproducibility concerns we try to have deterministic dataset generations.
Do you think we can simply assume that this file always exists ?
Otherwise we can keep it this way..

Reply:
Ok, changed it to make it deterministic.